redis/tests/test_bignum.rs

```diff
@@ -0,0 +1,120 @@
+use redis::{ErrorKind, FromRedisValue, RedisResult, ToRedisArgs, Value};
+use std::str::FromStr;
+
+#[test]
+fn test_rust_decimal() {
+    use rust_decimal::Decimal;
+
+    let content = "-79228162514264.337593543950335";
```

Comment:
the tests mostly repeat themselves - can the logic be moved to a function?

Reply:
Done